docs/make.bat

```diff
@@ -0,0 +1,263 @@
+@ECHO OFF

```

Comment:
It should have created a Makefile also, if I read the sphinx docs correctly. How are we supposed to use a bat file?